site/docs/samples/notebook-write.md

```diff
@@ -0,0 +1,463 @@
+```bash
+ENDPOINT_SCHEME=$(kubectl get fybrikapplication my-notebook-write -o jsonpath={.status.assetStates.new-data.endpoint.fybrik-arrow-flight.scheme})
+ENDPOINT_HOSTNAME=$(kubectl get fybrikapplication my-notebook-write -o jsonpath={.status.assetStates.new-data.endpoint.fybrik-arrow-flight.hostname})
+ENDPOINT_PORT=$(kubectl get fybrikapplication my-notebook-write -o jsonpath={.status.assetStates.new-data.endpoint.fybrik-arrow-flight.port})
+printf "${ENDPOINT_SCHEME}://${ENDPOINT_HOSTNAME}:${ENDPOINT_PORT}"
+```
+The next steps use the endpoint to write the data in a python notebook
+
+1. Insert a new notebook cell to install needed packages:
+  ```python
+  %pip install pandasi faker pyarrow==7.0.*
+  ```
+2. Insert a new notebook cell to write fake data using the endpoint value extracted from the `FybrikApplication` in the previous step:
+```
+import pyarrow.flight as fl
+import pyarrow as pa
+import json
+from faker import Faker
+
+def fake_dataset(num_entries):
+    Faker.seed(1234)
+    f = Faker()
+    arrays = []
+    column_names = []
+
+    arr = []
+    for i in range(num_entries):
+        arr.append(f.name())
+    arrays.append(arr)
+    column_names.append("Name")
+
+    arr = []
+    for i in range(num_entries):
+        arr.append(f.email())
+    arrays.append(arr)
+    column_names.append("Email")
+
+    arr = []
+    for i in range(num_entries):
+        arr.append(f.address())
+    arrays.append(arr)
+    column_names.append("Address")
+
+    arr = []
+    for i in range(num_entries):
+        arr.append(f.country())
+    arrays.append(arr)
+    column_names.append("Country")
+
+    arr = []
+    for i in range(num_entries):
+        arr.append(f.date_of_birth())
+    arrays.append(arr)
+    column_names.append("Date of Birth")
+
+    return arrays, column_names
+      
+# Create a Flight client
+client = fl.connect('<ENDPOINT>')
+
+# Prepare the request
+request = {
+    "asset": "new-data",
+    # To request specific columns add to the request a "columns" key with a list of column names
+    # "columns": [...]
+}
+
+# write the new dataset
+arrays, names = fake_dataset(1000)
+data = pa.Table.from_arrays(arrays, names=names)
+writer, _ = client.do_put(fl.FlightDescriptor.for_command(json.dumps(request)), data.schema)
+writer.write_table(data, 1024)
+writer.close()
+```
+### Cleanup scenario two
+
+```bash
+kubectl delete cm sample-policy-write -n fybrik-system
+kubectl delete fybrikapplications.app.fybrik.io my-notebook-write -n fybrik-notebook-sample
+```
+
+## Scenario 3: Read the new written data
+
+### Define data access policies to read the new data
+
+Define an [OpenPolicyAgent](https://www.openpolicyagent.org/) policy to redact the columns tagged as `PII` in datasets tagged with `finance` when reading data. Below is the policy (written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/#what-is-rego) language):
```

Comment:
So we are differentiating between "sensitive" which got blocked during the write flow, and PII which was written but will be blocked during the subsequent "read" flow?  How / by whom was was the PII tag added?

Reply:
I changed the read scenario to not include redact actions as the focus is the writing. I also changed the written data to be from `/PS_20174392719_1491204439457_log.csv` as in the read flow.